### PR TITLE
Sprint 4 task 19

### PR DESCRIPTION
**Pull request by vadymhrnk**

### CODE REVIEW

File: EasyRestTests/PageObjects/WaiterOrdersPage.cs [hunk 1]
```diff
@@ -22,21 +23,29 @@ public WaiterOrdersPage(IWebDriver webDriver)
...
 		public void ClickHistoryTab() => HistoryTab.Click();
+		[AllureStep("Click on show more button")]
 		public void ClickShowMoreButton() => ShowMoreButton.Click();
+		[AllureStep("Click on start order button")]
 		public void ClickStartOrderButton() => StartOrderButton.Click();
+		[AllureStep("Click on close order button")]
 		public void ClickCloseOrderButton() => CloseOrderButton.Click();
-	
+
+		[AllureStep("Gets web element with \"success\" text")]
```
Comment (Wallert) [inline EasyRestTests/PageObjects/WaiterOrdersPage.cs:41]: You can use single quotes, like 'success'

**Review by Wallert — APPROVED** [review]
Okey

**Comment by Sarhsian** [comment]
Ok

**Review by Sarhsian — COMMENTED** [review]
good

**Review by Sarhsian — APPROVED** [review]

**Review by MishkaBaloo — COMMENTED** [review]
Good

**Pull request merged**